<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew" >
     <property name="title" >
      <string>New</string>
     </property>
     <addaction name="actionIconSet" />
    </widget>
    <addaction name="menuNew" />
    <addaction name="separator" />
    <addaction name="actFromDir" />
    <addaction name="separator" />
    <addaction name="actExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>MainBar</string>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="statusTip" >
    <string>Exit from IconSet Editor</string>
   </property>
  </action>
  <action name="actionIconSet" >
   <property name="text" >
    <string>IconSet</string>
   </property>
  </action>
  <action name="actFromDir" >
   <property name="text" >
    <string>Create IconSet from Directory...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>298</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
